Controller/LogsAdminController.php

```diff
@@ -56,7 +59,13 @@ public function decryptEncryptAction(Request $request): Response
         if ($form->isSubmitted() && $form->isValid()) {
             $log     = $form->getData();
             $content = $log->getContent();
-            $results = $log->isDecryptAction() ? $this->getDecryptedResults($content) : $this->getEncryptedResult($content);
+            try {
+                $results = $log->isDecryptAction() ? $this->getDecryptedResults($content) : $this->getEncryptedResult($content);
+            } catch (EncryptionException $e) {
+                $message = $log->isDecryptAction() ? 'admin.logs.decrypt.error' : 'admin.logs.encrypt.error';
+
+                $form->addError(new FormError($this->trans($message, [], 'EkinoDataProtectionBundle')));
```

Comment:
Wrong because the validation was already done and `$form->isValid()` returns `true`, use the flash bag